scripts/unix/install_osslsigncode.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# This script will install the osslsigncode dependency (https://github.com/mtrojnar/osslsigncode) required to release this project
+
+PATH_STORAGE=/usr/local/bin
+
+# Create a temporary folder
+TEMP_FOLDER="temp"
+mkdir -p "$TEMP_FOLDER"
+cd "$TEMP_FOLDER" || exit
+
+# Download the latest macOS release
+# Courtesy: https://gist.github.com/steinwaywhw/a4cd19cda655b8249d908261a62687f8
+REPO="mtrojnar/osslsigncode"
+curl -s https://api.github.com/repos/$REPO/releases/latest \
+| grep "browser_download_url.*macOS.zip" \
+| cut -d : -f 2,3 \
+| tr -d \" \
+| wget -qi -
```

Comment:
thought: I don't love this, very brittle. We can roll with this for now since it'll only affect us but we should try to avoid clever hacks like this